lib/TabsNew/TabsBase.js

```diff
@@ -0,0 +1,19 @@
+import { func } from 'prop-types';
+
+export function TabsBase({ tabsLength, children }) {
+  const style = {
+    'flex-basis': 0,
+    'min-width': tabsLength >= 8 ? '12.5%' : 0,
+    'max-width': tabsLength >= 8 ? '12.5%' : 'initial'
+  };
+
+  return children('flex group flex-grow h-10', style);
```

Comment:
💚can we reasonably assume that a tab will always render a button and just plop a button here instead?

Reply:
Unfortunately we can't assume that. In our case they are going to be `react-router` `Link`s. This component isn't in gather-ui either! So I felt it made most sense to make it a render props component